### PR TITLE
refactor: align naming between libraries

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### CODE REVIEW

File: packages/showcases/core/csf/components/VtmnTextInput.csf.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  },
+  disabled: {
+    type: { name: 'boolean', required: false },
+    description: 'The disabled state of the input.',
+    defaultValue: false,
+    control: { type: 'boolean' },
+  },
+  valid: {
+    type: { name: 'boolean', required: false },
+    description: 'The validity state of the input.',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/VtmnTextInput.csf.js:51]:
```suggestion
    description: 'The valid state of the input.',
```

File: packages/showcases/core/csf/components/VtmnButton.csf.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+export const parameters = {
+  backgrounds: { default: 'grey' },
+  actions: {
+    handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+  },
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vtmn-Web?node-id=31%3A136',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/VtmnButton.csf.js:65]:
```suggestion
    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vitamin---Web-UI-kit-(v0)?node-id=1866%3A31',
```

File: packages/showcases/core/csf/components/VtmnLink.csf.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+export const parameters = {
+  backgrounds: { default: 'grey' },
+  actions: {
+    handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+  },
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vtmn---Web?node-id=1207%3A8898',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/VtmnLink.csf.js:46]:
```suggestion
    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vitamin---Web-UI-kit-(v0)?node-id=1207%3A8900',
```

File: packages/showcases/core/csf/components/VtmnTextInput.csf.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+};
+
+export const parameters = {
+  backgrounds: { default: 'grey' },
+  actions: {
+    handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+  },
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vtmn-Web?node-id=150%3A3410',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/VtmnTextInput.csf.js:70]:
```suggestion
    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vitamin---Web-UI-kit-(v0)?node-id=150%3A4893,
```

**Review by GaspardMathon — APPROVED** [review]

File: packages/showcases/svelte/stories/components/VtmnButton/VtmnButtonView.svelte [hunk 5]
```diff
@@ -4,6 +4,8 @@
    * @wrapper
    */
   import VtmnButton from '@vtmn/svelte/src/components/VtmnButton.svelte';
+
+  export let children = 'Button';
 </script>
 
-<VtmnButton {...$$restProps}>Button</VtmnButton>
+<VtmnButton {...$$restProps}>{children}</VtmnButton>
```
Comment (xavhan) [inline packages/showcases/svelte/stories/components/VtmnButton/VtmnButtonView.svelte:11]: is it really $$resProps you want or $$props would suffice ? 

https://svelte.dev/docs#Attributes_and_props

**Review by xavhan — COMMENTED** [review]
for my knowledge, what is CSF ?

**Comment by lauthieb** [comment]
@xavhan CSF is the acronym of "Component Story Format" introduced in Storybook 5.2: https://storybook.js.org/docs/react/api/csf

File: packages/showcases/svelte/stories/components/VtmnButton/VtmnButtonView.svelte (hunk 5 above)
Comment (lauthieb) [reply packages/showcases/svelte/stories/components/VtmnButton/VtmnButtonView.svelte:11]: Yes you're right! Thanks Xavier

**Pull request merged**